frontend/src/pages/approvers-tree/ApproversTree.jsx

```diff
@@ -354,7 +354,8 @@ const ApproversTree = () => {
           {selectedForm &&
             dataset.map(
               (aN, anI) =>
-                aN.children?.length > 0 && (
+                aN.children &&
+                aN.children.length > 0 && (
```

Comment:
why do we need to double-check them?
we can do `!!aN.children?.length && ()` isn't it?